pkg/services/app/remove.go

```diff
@@ -0,0 +1,56 @@
+package app
+
+import (
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	wego "github.com/weaveworks/weave-gitops/api/v1alpha1"
+)
+
+type AutomationManifestPaths struct { // source for automation isn't currently stored
+	AppDirAutomationPath    string
+	TargetDirAutomationPath string
+}
+
+func dirExists(d string) bool {
+	info, err := os.Stat(d)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return info.IsDir()
+}
+
+// findAppManifests locates all manifests associated with a specific application within a repository
+func findAppManifests(application wego.Application, repoDir string) ([][]byte, error) {
```

Comment:
it seems this function is only used in the tests, show it be move it there ? 🤔

Reply:
It's only used in tests right now (because I wrote the test for it first :-)). It will be used in the actual code later.